docker-compose.yaml

```diff
@@ -23,6 +30,23 @@ services:
       POSTGRES_DB: cla_backend
 
   # applications
+  cla_public:
+    image: ${ECR_ENDPOINT}/cla_public:develop
```

Comment:
💭 I recently got confused by this while looking at `cla_public`'s build job which got me thinking: it's probably better to expose this URL as it's a bit cryptic like this.

Reply:
Gotcha